zerver/models/users.py

```diff
@@ -116,6 +116,20 @@ class UserBaseSettings(models.Model):
 
     web_mark_read_on_scroll_policy = models.SmallIntegerField(default=MARK_READ_ON_SCROLL_ALWAYS)
 
+    # UI setting controlling if clicking on a channel link should open
+    # the channel feed (interleaved view) or narrow to the first topic
+    # in the channel.
+
+    NARROW_TO_FIRST_TOPIC_IN_CHANNEL = 1
+    NARROW_TO_CHANNEL_FEED = 2
```

Comment:
Let's call these something more prefixed, like `WEB_CHANNEL_DEFAULT_VIEW_FIRST_TOPIC` and `WEB_CHANNEL_DEFAULT_VIEW_CHANNEL_FEED`.

Later we'll convert these to formal enums to get rid of the really long names, but our current pattern is to namespace these with the setting name.

Reply:
Makes sense, done!